execution/evm/vm.go

```diff
@@ -493,6 +493,68 @@ func (vm *VM) call(caller acm.Account, callee acm.MutableAccount, code, input []
 			stack.Push64(int64(res))
 			vm.Debugf(" => 0x%X\n", res)
 
+		case SHL: //0x1B
+			shift, popErr := stack.PopU64()
+
+			if popErr != nil {
+				return nil, firstErr(err, popErr)
```

Comment:
According to EIP-145, any shift value > 256 should result in 0. This code returns an error if the shift value > 2^64.